examples/jsm/postprocessing/RenderTransitionPass.js

```diff
@@ -179,9 +157,6 @@ class RenderTransitionPass extends Pass {
 
 					}
 
-					#include <tonemapping_fragment>
-					#include <colorspace_fragment>
```

Comment:
Using `OutputPass` is mandatory so it's inconsistent if only `RenderTransitionPass` does tone mapping and color space conversion on its own.